FitGoal/FitGoal/Singup&Singin/AuthenticationViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  AuthenticationViewController.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 3/4/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class AuthenticationViewController: UIViewController {
```

Comment:
Remember to sort the items inside the class, first the variables and then the functions.